laPics/app/src/main/java/lapics/sergeybudkov/ru/lapics/AllPictureShow.java

```diff
@@ -0,0 +1,160 @@
+package lapics.sergeybudkov.ru.lapics;
+
+import android.app.Activity;
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.content.res.Configuration;
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.GridView;
+import android.widget.Toast;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AllPictureShow extends Activity {
+
+    public static int W,H;
+    private int width;
+    public static ProgressDialog dialog;
```

Comment:
it is very bad to make any UI element static, so its life is longer than life of bounded Activity